src/_includes/navbar.liquid

```diff
@@ -60,6 +60,8 @@
                 </div>
             </div>
 
+            {% include nav_link.liquid text="Openings" href="/openings/" %}
```

Comment:
Top nav items are prime real-estate, especially because having too many there makes it unusable on some screen sizes. I'd prefer you did what I asked before, which was to place it within the "more pages" dropdown and place a link to it on the front page somewhere.